### DISCUSSION TITLE
Rx.NET/System.Reactive 5.1/6.0 release?

### DISCUSSION THREAD

**Discussion by swythan** [dotnet/reactive#1781] [General]
Hi. I just wanted to check in to see if there is a plan to release an updated version of Rx.NET at any point?

I see that the [releases page](https://github.com/dotnet/reactive/releases) has an [Ix.NET 6.0.1](https://github.com/dotnet/reactive/releases/tag/ixnet-v6.0.1) release, and it actually lists a bugfix (https://github.com/dotnet/reactive/pull/1626) for Rx.NET that I'm interested in, but there isn't an equivalent Rx.NET release. 

The latest System.Reactive packages on nuget.org are all 5.0.0:
![image](https://user-images.githubusercontent.com/971734/174049803-3eec88c3-f01e-41b7-8edd-7f666dba51ef.png)

I see that [Rx.NET/Source/version.json](https://github.com/dotnet/reactive/blob/main/Rx.NET/Source/version.json) is still at `5.1.0-preview` so I suppose it's not just a release-process accident.

**Comment by bednar** [comment]
I am also interested about new version of `System.Reactive`.

**Comment by twtdev** [comment]
Definitely need this. My app is currently being held back to using .NET 5.0-windows10.0.19041 because of RX. It would be really nice to upgrade to .NET 6.0, if only RX for 6.0 would get released

**Comment by daveposn** [comment]
We're in the planning stages for a .NET application.  If we could just get a rough ETA for RX for 6.0 it would help us immensely,  Thanks for all your hard work.

**Comment by ken-okabe** [comment]
I also need Rx.NET6.0.

It's been quite a while since .net6 was released. Thanks.

**Comment by swythan** [comment]
Ah. The discussion in this issue is enlightening:
https://github.com/dotnet/reactive/issues/1822

Looks like we need a new maintainer.

**Comment by swythan** [comment]
Looks like endjin may be stepping in. Hurrah!

See https://github.com/dotnet/reactive/pull/1879

**Comment by HowardvanRooijen** [comment]
📢 Rx.NET v6.0 is now available: https://endjin.com/blog/2023/05/rx-dotnet-v6-released

**Reply by swythan** [reply]
Many thanks to you @HowardvanRooijen and of course to Endjin for taking on maintainership.
